paddle/fluid/framework/operator.cc

```diff
@@ -1114,9 +1118,13 @@ Scope* OperatorWithKernel::PrepareData(
       // If this op is not called by an Executor or ParallelExecutor, it should
       // called by a NaiveExecutor, the NaiveExecutor will cache the scopes and
       // variables, that behavior a lot different.
-      if (!run_by_executor_) {
+      enable_transferscope_cache = false;
```

Comment:
Please add more comments on why only works on `is_gpu_place`, you can paste the issue, and add something like`Have a discussion with @Luotao for cpu inference`

Reply:
done